docs/api/api.md

```
@@ -1,212 +1,84 @@
-- [Base REST Controller](#base-rest-controller)
-- [1. Role-Specific Controllers.](#1-role-specifix-controllers)
-    - [Admin REST Controller.](#admin-rest-controller)
-    - [Vendor REST Controller.](#vendor-rest-controller)
-    - [Customer REST Controller.](#customer-rest-controller)
+- [Creating a REST Controller](#creating-a-rest-controller)
+  - [Extending a Dokan Base Controller](#extending-a-dokan-base-controller)
+  - [Extending Other Controllers](#extending-other-controllers)
+- [VendorAuthorizable Trait](#vendorauthorizable-trait)
+- [Example Implementations](#example-implementations)
+  - [Admin Controller Example](#admin-controller-example)
+  - [Vendor Controller Example](#vendor-controller-example)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)